Find a functionally equivalent code:android.util.ArrayMap.removeAt:COMMENT
Method Modifier: public      
Comment:/**
 * Remove the key/value mapping at the given index.
 * @param index The desired index, must be between 0 and {@link #size()}-1.
 * @return Returns the value that was stored at this index.
 */

@@ -5,12 +5,12 @@
         if (DEBUG)
             Log.d(TAG, "remove: shrink from " + mHashes.length + " to 0");
         freeArrays(mHashes, mArray, mSize);
-        mHashes = ContainerHelpers.EMPTY_INTS;
-        mArray = ContainerHelpers.EMPTY_OBJECTS;
+        mHashes = EmptyArray.INT;
+        mArray = EmptyArray.OBJECT;
         mSize = 0;
     } else {
         if (mHashes.length > (BASE_SIZE * 2) && mSize < mHashes.length / 3) {

